### PR TITLE
asionet 1.5.0

### PR DESCRIPTION
**Pull request by rtosman**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/asionet/config.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+versions:
+  "1.5.0":
+  folder: all
```
Comment (madebr) [inline recipes/asionet/config.yml:3]:
```suggestion
    folder: all
```

**Review by madebr — CHANGES_REQUESTED** [review]
This is missing a test package.
Please include a small cxx source using the asionet headers + build it.

**Comment by rtosman** [comment]
> This is missing a test package.
> Please include a small cxx source using the asionet headers + build it.

Sorry, I am new to conancenter. Is the test supposed to be part of the library or part of the recipe?

I have a completely separate project that is the test for this library, but I was going to submit that once this was in, since it has a dependency on this library (obviously).

**Comment by ruilvo** [comment]
> Sorry, I am new to conancenter. Is the test supposed to be part of the library or part of the recipe?
> 
> I have a completely separate project that is the test for this library, but I was going to submit that once this was in, since it has a dependency on this library (obviously).

My advice is for you to look at the other recipes :) The best place to start is to really use another recipe that's like yours, in this case, any that has "header_only" on it :) You'll see what is expected.

File: recipes/asionet/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, tools
+
+class ASIOnetConan(ConanFile):
+    name = "asionet"
+    version = "1.5.0"
```
Comment (ruilvo) [inline recipes/asionet/all/conanfile.py:5]: Please remove the version from the python script so that you can reuse it for other versions in the future.

**Review by ruilvo — CHANGES_REQUESTED** [review]

**Comment by ruilvo** [comment]
Please use more descriptive titles in PR submissions. And please fill out the fields in the PR creation:

> Specify library name and version: lib/1.0
> 
> This is also a good place to share with all of us why you are submitting this PR (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

This is really useful information.

**Pull request closed without merging**